bindings/nodejs/lib/types/client/network.ts

```diff
@@ -49,6 +49,8 @@ export interface INode {
     auth?: IAuth;
     /** Whether the node is disabled or not. */
     disabled?: boolean;
+    /** Whether the node is a permanode or not. */
+    permanode?: boolean;
```

Comment:
Does it not need to be `Option<>` in Rust to be equivalent to `?` ? Or does `#[serde(default)]` achieve the same?